spark/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -25,6 +25,7 @@ import java.util.Locale
 import scala.sys.process.Process
 
 // scalastyle:off import.ordering.noEmptyLine
+// scalastyle:off line.size.limit
```

Comment:
This seems unnecessary when only deleting stuff?

Reply:
Ah, but then, don't add it at the top, but just scope it where it's needed, please? Or is just sooooo many places that that would be crazy?

Reply:
I am afraid there are 38 errors :(.

Reply:
That are all non-consecutive?

Reply:
I am afraid they are not.